lib/nolibc/string.c

```diff
@@ -815,3 +815,8 @@ char *strncat(char *dest, const char *src, size_t n)
 	*dest++ = 0;
 	return a;
 }
+
+int bcmp(const void *s1, const void *s2, size_t count)
```

Comment:
As far as I saw, `bcmp` is an obsolete libc function and `memcmp` should be used instead. Where is `bcmp` still used?

Reply:
rustc generate code that uses bcmp. LLVM 9.0 still emits calls to `bcmp`

Reply:
Hum, okay. Then I wonder two points:
1) Could we introduce an `alias` to `memcmp` instead of defining a new function?
2) I am not yet sure what the best location for the alias definition would be. Do you know if `musl` provides `bcmp`? If yes, it has to be `nolibc`. If not, we may want to place the alias with the rust support library that uses `cargo`.

Reply:
Yes, musl 1.1.19 (the version ported on unikraft) defines bcmp at `src/string/bcmp.c`. They use the same call to `memcpy`, so I'd keep this as a function and not an alias to match musl's approach.